auth0_flutter/lib/src/web/credentials_extension.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:auth0_flutter_platform_interface/auth0_flutter_platform_interface.dart';
+
+import 'js_interop.dart';
+import 'jwt_decode.dart';
+import 'user_profile_extension.dart';
+
+extension CredentialsExtension on Credentials {
+  static Credentials fromWeb(final WebCredentials webCredentials) {
+    final expiresIn = webCredentials.expires_in as int;
+    final expiresAt = DateTime.now().add(Duration(seconds: expiresIn));
+    final claims = JWT.decode(webCredentials.id_token);
+    final user = UserProfileExtension.fromClaims(claims);
+
+    return Credentials(
+        idToken: webCredentials.id_token,
+        accessToken: webCredentials.access_token,
+        expiresAt: expiresAt,
+        user: user,
+        tokenType: 'Bearer');
```

Comment:
I believe we get `tokenType` from the `getTokenSilently` detailed response - should we use that, or hard code it?